### DISCUSSION TITLE
[Documentation] equity vs. equity35 vs. equity65 vs. unlisted_equity

### DISCUSSION THREAD

**Discussion by the-solipsist** [ananthakumaran/paisa#171] [General]
The documentation at https://paisa.fyi/reference/tax/tax-harvesting/ and https://paisa.fyi/reference/config/ only contains 4 categories of equity mutual funds
- debt
- equity35
- equity65
- unlisted_equity

However, the dropdown menu in Paisa provides for 4 options:
- debt
- equity
- equity35
- equity65
- unlisted_equity

The documentation should be updated to clarify what 'equity' is for.

Do note: Since 2018, mutual funds cannot invest in unlisted equity.

Also, the documentation seems to implicitly suggest that for `debt` category mutual funds, a tax harvesting period of 1095 days works out, whereas this isn't true after March 1, 2023.

**Comment by ananthakumaran** [comment]
> equity

equity is deprecated, it was there before the 35/65 rule was introduced. I didn't remove it as it would be a breaking change (config wise), but it's just an alias for equity65

> Do note: Since 2018, mutual funds cannot invest in unlisted equity.

> Also, the documentation seems to implicitly suggest that for debt category mutual funds, a tax harvesting period of 1095 days works out, whereas this isn't true after March 1, 2023.

I did update the code to handle this taking grandfathering into account.

In general, I am not going down this route anymore. I introduced [sheets](https://paisa.fyi/reference/sheets/) as a generic replacement and will remove the tax section and related functionalities after some time.
